### PR TITLE
[auto-fix] interface type updated for Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge

### PR DESCRIPTION
**Pull request by gurbaj5124871**
**This is an automated generated pr**
**changelog**
- auto-fix: interface type updated for Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge
    
**Block Data**
network: osmosis-1
height: 13117454

### CODE REVIEW

File: src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts [hunk 1]
```diff
@@ -920,25 +920,29 @@ export interface Osmosis1TrxMsgOsmosisIncentivesMsgAddToGauge
...
-      denom: string;
-      duration: string;
-      timestamp: string;
-    };
     numEpochsPaidOver: string;
-  };
+    poolId: string;
+}
+interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo {
+    lockQueryType: string;
```
Comment (chriskscan) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:936]: `denom` is an optional string here

Comment (chriskscan) [reply src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:936]: This is the proto:
message QueryCondition {
  LockQueryType lock_query_type = 1;
  // Denom represents the token denomination we are looking to lock up
  string denom = 2;
  google.protobuf.Duration duration = 3;
  google.protobuf.Timestamp timestamp = 4;
}

File: src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts
Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 13325789
data:
```
[
  {
    "owner": "osmo1dlwqwzggptdvx9rewhca2axz64klpzw65ttuvp",
    "distributeTo": {
      "lockQueryType": "NoLock",
      "duration": "0s",
      "timestamp": "0001-01-01T00:00:00Z"
    },
    "coins": [
      {
        "denom": "ibc/A0CC0CF735BFB30E730C70019D4218A1244FF383503FF7579C9201AB93CA9293",
        "amount": "15000000000"
      }
    ],
    "startTime": "2024-01-19T00:01:32Z",
    "numEpochsPaidOver": "31",
    "poolId": "1101"
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[29].messages[0].data.distributeTo.denom",
    "expected": "string"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge {
    type: string;
    data: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData {
    owner: string;
    distributeTo: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo;
    coins: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem[];
    startTime: string;
    numEpochsPaidOver: string;
    poolId: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo {
    lockQueryType: string;
    duration: string;
    timestamp: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem {
    denom: string;
    amount: string;
}

```

Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 13707809
data:
```
[
  {
    "owner": "osmo1dlwqwzggptdvx9rewhca2axz64klpzw65ttuvp",
    "distributeTo": {
      "lockQueryType": "NoLock",
      "duration": "0s",
      "timestamp": "0001-01-01T00:00:00Z"
    },
    "coins": [
      {
        "denom": "ibc/8061A06D3BD4D52C4A28FFECF7150D370393AF0BA661C3776C54FF32836C3961",
        "amount": "200000000000000000000000"
      }
    ],
    "startTime": "2024-02-12T00:00:01Z",
    "numEpochsPaidOver": "31",
    "poolId": "1323"
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[0].messages[0].data.distributeTo.denom",
    "expected": "string"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge {
    type: string;
    data: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData {
    owner: string;
    distributeTo: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo;
    coins: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem[];
    startTime: string;
    numEpochsPaidOver: string;
    poolId: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo {
    lockQueryType: string;
    duration: string;
    timestamp: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem {
    denom: string;
    amount: string;
}

```

Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 16391150
data:
```
[
  {
    "owner": "osmo1k7ehz7s84k03ghr2ea06jlmteju74pg4w8zqt4",
    "distributeTo": {
      "lockQueryType": "NoLock",
      "duration": "3600s",
      "timestamp": "0001-01-01T00:00:00Z"
    },
    "coins": [
      {
        "denom": "ibc/903A61A498756EA560B85A85132D3AEE21B5DEDD41213725D22ABF276EA6945E",
        "amount": "10000000000"
      }
    ],
    "startTime": "2024-06-04T11:00:00Z",
    "numEpochsPaidOver": "30",
    "poolId": "1431"
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[2].messages[0].data.distributeTo.denom",
    "expected": "string"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge {
    type: string;
    data: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData {
    owner: string;
    distributeTo: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo;
    coins: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem[];
    startTime: string;
    numEpochsPaidOver: string;
    poolId: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo {
    lockQueryType: string;
    duration: string;
    timestamp: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem {
    denom: string;
    amount: string;
}

```

Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 17443471
data:
```
[
  {
    "owner": "osmo1k7ehz7s84k03ghr2ea06jlmteju74pg4w8zqt4",
    "distributeTo": {
      "lockQueryType": "NoLock",
      "duration": "3600s",
      "timestamp": "0001-01-01T00:00:00Z"
    },
    "coins": [
      {
        "denom": "ibc/903A61A498756EA560B85A85132D3AEE21B5DEDD41213725D22ABF276EA6945E",
        "amount": "10000000000"
      }
    ],
    "startTime": "2024-07-03T20:00:00Z",
    "numEpochsPaidOver": "30",
    "poolId": "1431"
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[1].messages[0].data.distributeTo.denom",
    "expected": "string"
  },
  {
    "path": "$input.transactions[1].messages[0].data.poolId",
    "expected": "undefined",
    "value": "1431"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge {
    type: string;
    data: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData {
    owner: string;
    distributeTo: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo;
    coins: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem[];
    startTime: string;
    numEpochsPaidOver: string;
    poolId: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo {
    lockQueryType: string;
    duration: string;
    timestamp: string;
}
interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem {
    denom: string;
    amount: string;
}

```

**Pull request merged**